client/src/modules/templates/bhCard.tmpl.html

```diff
@@ -0,0 +1,23 @@
+<div class="panel panel-default bh-card" data-group-entry="{{card.id}}">
+  <div class="panel-body">
+    <div class="row">
+      <div class="col-md-6">
+        <h4 style="margin-top : 0px;" ng-transclude="title"></h4>
+        <h5 style="margin-top : 0px;" ng-transclude="subtitle"></h5>
+      </div>
+      <div class="col-md-6 text-float-util">
```

Comment:
What is `text-float-util`?  I think this is first time I've seen it!

Reply:
Apparently and bizarrely it sets `text-align: right`. `text-float-util` is just a ruse to make the programmer think this is more important than it :disappointed_relieved:. 

It looks like this is a carry over from older code, I will _consider_ using the more aptly named bootstrap `text-right`.